feature-governance-api/src/main/java/io/novafoundation/nova/feature_governance_api/domain/tindergov/TinderGovInteractor.kt

```diff
@@ -10,7 +15,19 @@ interface TinderGovInteractor {
 
     fun observeReferendaAvailableToVote(coroutineScope: CoroutineScope): Flow<List<ReferendumPreview>>
 
+    fun observeTinderGovBasket(): Flow<List<TinderGovBasketItem>>
+
+    suspend fun addItemToBasket(referendumId: ReferendumId, voteType: VoteType)
+
     suspend fun loadReferendumSummary(id: ReferendumId): String?
 
     suspend fun loadReferendumAmount(referendumPreview: ReferendumPreview): BigInteger?
+
+    suspend fun setVotingPower(chainId: ChainId, amount: BigInteger, conviction: Conviction)
+
+    suspend fun getVotingPower(metaId: Long, chainId: ChainId): VotingPower?
```

Comment:
Do we actually need fields `metaId` and `chainId` in `VotingPower` if we're not passing `VotingPower` to `setVotingPower` and not using them later on? 

Imho we should either remove those fields from voting power or to make `setVotingPower` accept `VotingPower` directly